### PR TITLE
JWWEB-987 Update api errors management

### PR DESCRIPTION
**Pull request by IViol**

### CODE REVIEW

File: src/services/blockExplorer.js [hunk 1]
```diff
@@ -265,12 +239,31 @@ function getETHTransactions(
...
+
+  const params: BlockExplorerAPIParams = {
     endblock,
     startblock,
     sort: 'desc',
     action: 'txlist',
-  }, networkId, owner)
+  }
+
+  const queryParams: string = Object
```
Comment (stcherenkov) [inline src/services/blockExplorer.js]: There is a good lib called [qs](https://www.npmjs.com/package/qs) for that. It also handles edge cases and has great test coverage. Take a look.

File: src/services/callAPI.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  }).catch(() => handleRequestError(
+    requestInfo,
+    options,
+    params,
+    retryCount,
+  )).then((response: Response): Promise<any> => {
+    if (response.ok) {
+      return response.json()
+    }
+
```
Comment (stcherenkov) [inline src/services/callAPI.js]: Let's also send exception to GA with future migration to Sentry. Docs on GA exceptions handling are here: https://developers.google.com/analytics/devguides/collection/analyticsjs/exceptions
